public/styles/components/sidebar/_sidebar.scss

```diff
@@ -105,6 +105,11 @@
 
   padding: 5px 40px;
 
+  cursor: pointer;
```

Comment:
I've done the same in my PR ;) https://github.com/guardian/campaign-central/pull/45
Will get rid of it.